webpack/ForemanColumnExtensions/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+
+const RecommendationsCell = hostDetails => {
+  const insightsAttributes = propsToCamelCase(
+    // eslint-disable-next-line camelcase
+    hostDetails?.insights_attributes ?? {}
+  );
+  // Local insights advisor
+  if (insightsAttributes.useLocalAdvisorEngine) {
+    // TODO: Replace this placeholder with the actual local advisor integration
+    return <span>Local advisor placeholder</span>;
+  }
+
+  // Hosted insights advisor
+  const hitsCount = insightsAttributes.insightsHitsCount;
```

Comment:
What do you think of? `const { insightsHitsCount: hitsCount } = insightsAttributes;`